app/javascript/components/requests/Receipt.jsx

```diff
@@ -16,42 +17,14 @@ class Receipt extends React.Component {
     };
   }
 
-  getAttr = (request) => {
-    let attr = {
-      "Price": "$" + convertToCurrency(request.receipt.price),
-      "Purchase Date": new Date(request.receipt.purchase_date).toLocaleDateString(),
-      "Transaction Type": request.receipt.transaction_type,
-      "Description": request.receipt.comment
```

Comment:
Can we bring back `request.receipt.comment` somewhere? Maybe underneath the patron's message?

Reply:
![image](https://user-images.githubusercontent.com/23442055/55921751-ed985300-5bb2-11e9-8784-2779c63b26da.png)
 something like this?

Reply:
yes plese